src/main/java/org/wildfly/security/auth/util/GSSCredentialSecurityFactory.java

```diff
@@ -64,18 +65,6 @@
     private static final String KRB5LoginModule = "com.sun.security.auth.module.Krb5LoginModule";
     private static final String IBMKRB5LoginModule = "com.ibm.security.auth.module.Krb5LoginModule";
 
-    public static final Oid KERBEROS_V5;
-    public static final Oid SPNEGO;
-
```

Comment:
Unfortunately this package has been flagged as public API, as this class is also public we can't remove public static fields from it - if this is not the best place for the fields I would suggest marking them as deprecated so we can review removing them at a later point.

Reply:
ok, this is not the worst place, will keep this instead of OidUtils...